src/DssSpell.sol

```diff
@@ -49,82 +121,187 @@ contract DssSpellAction is DssAction {
+            D3MHubLike(D3M_HUB).file("end", DssExecLib.end());
 
-    function actions() public override {
-        // -------------------- Update RWA007, RWA008, RWA009 Legal Documents ---------------------
-        // https://forum.makerdao.com/t/nov-16-2022-executive-contents/18747
-        //
-        // Monetalis (RWA007-A) legal update doc
-        _updateDoc("RWA007-A", "QmejL1CKKN5vCwp9QD1gebnnAM2MJSt9XbF64uy4ptkJtR");
-        // SG Forge OFH (RWA008-A) legal update doc
-        _updateDoc("RWA008-A", "QmZ4heYjptvj3ovafADJpXYMFXMyY3yQjkTXpvjFPnAKcy");
-        // HVB (RWA009-A) legal update doc
-        _updateDoc("RWA009-A", "QmeRrbDF8MVPQfNe83gWf2qV48jApVigm1WyjEtDXCZ5rT");
-
-        // RWA007-A autoline changes:
-        // - bump oracle price to 500m
-        // - increase DC to 500m
-        // - increase autoline `gap` to 100m
-        //
-        // https://vote.makerdao.com/polling/QmSfMtTM#poll-detail
-        DssExecLib.setIlkAutoLineParameters(
-            "RWA007-A",
-            500 * MILLION,
-            100 * MILLION,
-            1 weeks
-        );
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
-            "RWA007-A",
-             500 * MILLION * WAD
-        );
-        DssExecLib.updateCollateralPrice("RWA007-A");
-
-        // RETH-A autoline changes:
-        // - line 5m
-        // - gap 3m
-        // - ttl 8h
-        //
-        //  https://vote.makerdao.com/polling/QmfMswF2#poll-detail
-        DssExecLib.setIlkAutoLineParameters(
-            "RETH-A",
-            5 * MILLION,
-            3 * MILLION,
-            8 hours
-        );
-
-        // -------------------- Changelog Update & housekeeping ---------------------
-
-        // - Change "RWA007_A_INPUT_CONDUIT_URN" to "RWA007_A_INPUT_CONDUIT"
-        // - Change "RWA007_A_INPUT_CONDUIT_JAR" to "RWA007_A_JAR_INPUT_CONDUIT"
-        //
-        DssExecLib.setChangelogAddress("RWA007_A_INPUT_CONDUIT", DssExecLib.getChangelogAddress("RWA007_A_INPUT_CONDUIT_URN"));
-        DssExecLib.setChangelogAddress("RWA007_A_JAR_INPUT_CONDUIT", DssExecLib.getChangelogAddress("RWA007_A_INPUT_CONDUIT_JAR"));
-        ChainlogLike(DssExecLib.LOG).removeAddress("RWA007_A_INPUT_CONDUIT_URN");
-        ChainlogLike(DssExecLib.LOG).removeAddress("RWA007_A_INPUT_CONDUIT_JAR");
-
-        // Bump version
-        DssExecLib.setChangelogVersion("1.14.5");
+            D3MMomLike(D3M_MOM).setAuthority(DssExecLib.getChangelogAddress("MCD_ADM"));
+
+            D3MCompoundPoolLike(D3M_COMPOUND_POOL).file("king", address(this));
+
+            D3MCompoundPlanLike(D3M_COMPOUND_PLAN).rely(D3M_MOM);
+            D3MCompoundPlanLike(D3M_COMPOUND_PLAN).file("barb", D3M_COMP_BORROW_RATE);
+
+            D3MOracleLike(D3M_ORACLE).file("hub", D3M_HUB);
+
+            spot.file(ILK, "pip", D3M_ORACLE);
+            spot.file(ILK, "mat", RAY);
+            vat.rely(D3M_HUB);
+            vat.init(ILK);
+            JugAbstract(DssExecLib.jug()).init(ILK);
```

Comment:
Let's call `drip` after, so `rate` gets set.

Reply:
In regular collateral onboarding this happens here: `setIlkStabilityFee`

Reply:
Rate is set in `vat.init()`, no?